pydantic_core/core_schema.py

```diff
@@ -93,6 +93,12 @@ def __repr__(self) -> str:
         ...
 
 
+class FieldSerializationInfo(SerializationInfo, Protocol):
+    @property
+    def field_name(self) -> str:
```

Comment:
Is this the only thing on the info object when serializing fields? Surprising to me that we are introducing an info object for just one `str`, but I guess this will become a nice place to add other info in the future.

If there are other fields that will be present on it, I would be inclined to add them (for documentation-like purposes).

It makes sense to keep it like this given we might want to add other stuff in the future, if there's other fields now though I am wondering if it makes sense to add them

Reply:
They're there, look at the superclass.

Reply:
Looks like there are some other attributes (noticed this below):
```rust
        d.set_item("mode", self.mode(py))?;
        d.set_item("by_alias", self.by_alias)?;
        d.set_item("exclude_unset", self.exclude_unset)?;
        d.set_item("exclude_defaults", self.exclude_defaults)?;
        d.set_item("exclude_none", self.exclude_none)?;
        d.set_item("round_trip", self.round_trip)?;
```

Reply:
Didn't realize SerializationInfo defined all this stuff. 🤦